### DISCUSSION TITLE
https://www.irvanma.eu.org/blog/setting-up-ssh-github

### DISCUSSION THREAD

**Discussion by giscus** [irvanmalik48/realm#26] [Comments]
# https://www.irvanma.eu.org/blog/setting-up-ssh-github

Git gud, folks!

https://www.irvanma.eu.org/blog/setting-up-ssh-github

**Comment by KenHV** [comment]
Thanks for the tip about using SSH keys to sign commits :)

**Reply by irvanmalik48** [reply]
ywyw
